packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -527,15 +561,15 @@ class GutterItem extends StatelessWidget {
 class Lines extends StatefulWidget {
   const Lines({
     Key key,
-    @required this.constraints,
+    @required this.height,
     @required this.scrollController,
     @required this.lines,
     @required this.pausedFrame,
     @required this.searchMatchesNotifier,
     @required this.activeSearchMatchNotifier,
   }) : super(key: key);
 
-  final BoxConstraints constraints;
+  final num height;
```

Comment:
flutter style: prefer to always be explicit about using `double` rather than `num` for num that can be a double. that is better for performance and avoids gotchas where an implicit cast from a num to double for a value that is an int could cause problems.

Reply:
Makes sense, switched to double